lib/commands/publish.js

```diff
@@ -41,17 +42,24 @@ module.exports = {
       broker: process.env.EMBER_CLI_PACT_BROKER,
       brokerUsername: process.env.EMBER_CLI_PACT_BROKER_USERNAME,
       brokerPassword: process.env.EMBER_CLI_PACT_BROKER_PASSWORD,
+      brokerToken: process.env.EMBER_CLI_PACT_BROKER_TOKEN,
       tags: process.env.EMBER_CLI_PACT_TAGS
     };
-    let mergedConfig = Object.assign({}, config, commandLineOptions);
```

Comment:
This was causing issues where by the default values from the `commandLineOptions` were overriding values from the environment. See PR description